helm/cluster-cloud-director/values.yaml

```
@@ -9,6 +9,135 @@ cluster:
+                        - set-networkd-units.service
+                  enabled: true
+                  name: static-routes.service
+                - contents:
+                    install:
+                      wantedBy:
+                        - multi-user.target
+                    service:
+                      environment:
+                        - OUTPUT=/run/metadata/coreos
+                      execStart:
+                        - /usr/bin/mkdir --parent /run/metadata
+                        - /usr/bin/bash -cv 'echo "COREOS_CUSTOM_HOSTNAME=$("$(find /usr/bin /usr/share/oem -name vmtoolsd -type f -executable 2>/dev/null | head -n 1)" --cmd "info-get guestinfo.ignition.vmname")" > ${OUTPUT}'
+                        - /usr/bin/bash -cv 'echo "COREOS_CUSTOM_IPV4=$(ip -4 addr show ens192 | awk \'/inet / {print $2}\' | cut -d \'/\' -f1)" >> ${OUTPUT}'
+                      remainAfterExit: "yes"
+                      restart: on-failure
+                      type: oneshot
+                    unit:
+                      after:
+                        - set-networkd-units.service
+                      description: Provides CoreOS metadata for dependent services.
+                  enabled: true
+                  name: coreos-metadata.service
+                - contents:
+                    install:
+                      wantedBy:
+                        - multi-user.target
+                    service:
+                      environmentFile:
+                        - /run/metadata/coreos
+                      execStart:
+                        - /opt/bin/set-hostname.sh
+                      remainAfterExit: "yes"
+                      type: oneshot
+                    unit:
+                      after:
+                        - coreos-metadata.service
+                      before:
+                        - teleport.service
+                      description: Set machine hostname
+                      requires:
+                        - coreos-metadata.service
+                  enabled: true
+                  name: set-hostname.service
+                - contents:
+                    install:
+                      wantedBy:
+                        - default.target
+                    service:
+                      execStart:
+                        - /usr/sbin/ethtool -K ens192 tx-udp_tnl-csum-segmentation off
+                        - /usr/sbin/ethtool -K ens192 tx-udp_tnl-segmentation off
+                      remainAfterExit: "yes"
+                      type: oneshot
+                    unit:
+                      after:
+                        - network.target
+                      description: Disable TCP segmentation offloading
+                  enabled: true
+                  name: ethtool-segmentation.service
+                - dropins:
+                    - contents: |-
+                        [Unit]
+                        Upholds=set-static-routes.service
```

```suggestion
                        Upholds=static-routes.service
```
You changed the name of the static routes unit